teams/types.toml

```
@@ -39,3 +39,6 @@ extra-people = ["eholk"]
 
 [permissions]
 bors.chalk.review = true
+perf = true
```

```suggestion
bors.rust.try = true
perf = true
```

Also need to give try permissions.